<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenOtherDialog</class>
 <widget class="QDialog" name="OpenOtherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Other</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>P&amp;reset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>presetCombo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="presetCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savePresetButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePresetButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="methodTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="networkTab">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>&amp;URL</string>
               </property>
               <property name="buddy">
                <cstring>urlLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="urlLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="decklinkTab">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="DecklinkProducerWidget" name="decklinkWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="colorTab">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="ColorProducerWidget" name="colorWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="noiseTab"/>
         <widget class="QWidget" name="isingTab">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="IsingWidget" name="isingWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="lissajousTab">
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="LissajousWidget" name="lissajousWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="plasmaTab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>10</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="PlasmaWidget" name="plasmaWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="v4lTab">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="Video4LinuxWidget" name="v4lWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlasmaWidget</class>
   <extends>QWidget</extends>
   <header>widgets/plasmawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LissajousWidget</class>
   <extends>QWidget</extends>
   <header>widgets/lissajouswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IsingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/isingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Video4LinuxWidget</class>
   <extends>QWidget</extends>
   <header>widgets/video4linuxwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorProducerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/colorproducerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DecklinkProducerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/decklinkproducerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenOtherDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenOtherDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
